src/core/sys/posix/stdio.d

```diff
@@ -344,3 +344,7 @@ unittest
     assert(memcmp(ptr, testdata.ptr, testdata.length*wchar_t.sizeof) == 0);
     assert(fclose(f) == 0);
 }
+
+
+ssize_t getdelim (char** lineptr, size_t* n, int delimiter, FILE* stream);
+ssize_t getline (char** lineptr, size_t* n, FILE* stream);
```

Comment:
```
ssize_t getdelim(scope char** lineptr, scope size_t* n, int delimiter, FILE* stream);
ssize_t getline(scope char** lineptr, scope size_t* n, FILE* stream);
```

Adding `scope` will allow passing pointers to local variables in `@safe` code. See also #1740, #1748 and #1749.